CHANGELOG.md

```diff
@@ -1,5 +1,24 @@
 ## v1.3.0 [unreleased]
 
+### Release Notes
+
+#### Continuous Query Statistics
+
+When enabled, each time a continuous query is completed, a number of details regarding the execution are written to the `cq_query` measurement of the internal monitor database (`_internal` by default). The tags and fields of interest are
+
+| tag / field | description                                           |
+|:----------- |:----------------------------------------------------- |
+| `db`        | name of database                                      |
+| `cq`        | name of continuous query                              |
+| `duration`  | query execution time in nanoseconds                   |
```

Comment:
Other stats use `durationNs` in the field name.  I'd suggest that for consistency.